pkgs/applications/misc/nimbo/default.nix

```
@@ -12,13 +13,20 @@ buildPythonApplication rec {
     rev = "v${version}";
     sha256 = "1fs28s9ynfxrb4rzba6cmik0kl0q0vkpb4zdappsq62jqf960k24";
   };
-
-  propagatedBuildInputs = [ setuptools boto3 awscli requests click pyyaml pydantic ];
+  nativeBuildInputs = [ installShellFiles ];
+  propagatedBuildInputs =
+    [ setuptools boto3 awscli requests click pyyaml pydantic ];
```

```suggestion
  propagatedBuildInputs = [ setuptools boto3 awscli requests click pyyaml pydantic ];
```

also makes the diff more readable.